tutorials/3d/mesh_lod.rst

```diff
@@ -133,9 +133,9 @@ camera moves away, resulting in higher performance but lower quality.
 
 If you need to perform per-object adjustments to mesh LOD, you can adjust how
 aggressive LOD transitions should be by adjusting the **LOD Bias** property on
-any node that inherits from GeometryInstance3D. Positive values will make LOD
-transitions happen sooner than usual (resulting in lower quality but higher
-performance). Negative values will make LOD transitions happen earlier than
+any node that inherits from GeometryInstance3D. Values *above* ``1.0`` will make
+LOD transitions happen sooner than usual (resulting in lower quality but higher
```

Comment:
```suggestion
LOD transitions happen sooner than usual (resulting in higher quality but lower
```

Reply:
The currently phrasing is already correct. Lower values of LOD Bias makes meshes look *better* at a given distance, not worse.

Reply:
The current behaviour is that values between 0 and 1 make the LOD transition happen sooner resulting in worse quality for the same distance. 

Values above one delay the LOD transition so you get higher quality at the same distance.

Reply:
> The current behaviour is that values between 0 and 1 make the LOD transition happen sooner resulting in worse quality for the same distance.
> 
> Values above one delay the LOD transition so you get higher quality at the same distance.

So the LOD Bias property works the opposite of the Mesh LOD Threshold project setting? I consider that to be pretty odd :slightly_frowning_face:

Reply:
> So the GeometryInstance3D LOD Bias property works the opposite of the Mesh LOD Threshold project setting? I consider that to be pretty odd slightly_frowning_face
>
>In general, I expect LOD bias to work in a fashion where positive values use lower-than-needed LODs and negative values use higher-than-needed LODs (similar to mipmap LOD bias).

Yes, it is rather odd. I would expect it to work like textureLod in GLSL as well. i.e. a bias of one bumps you up one level while -1 bumps you down one level

Reply:
Indeed, I've tested this on 4.0.beta11 and what you're saying is correct.

Could we change this behavior before release? It's likely to be a pitfall people will run into.

Reply:
I don't know. I'm not familiar enough with the LOD system to comment one why it was implemented that way